parquet_schemas/autopush.autopush.docker.autopush/conn.cfg

```diff
@@ -0,0 +1,35 @@
+-- vim:ft=lua
+message_matcher = message_matcher .. " && Fields[Fields.router_key] == 'webpush'"
+
+s3_path_dimensions = {
+	{name="log", source="Type"},
+	{name="router_key", source="Fields[Fields.router_key]"},
+	{name="date", source="Fields[Date]"},
+	{name="hour", source="Fields[Hour]"},
+}
+
+encoders_logging_common = {user_agent_transforms={["Fields.user_agent"]="Fields.browser_"}
+
+parquet_schema = [=[
+message Log {
+    required int64 Timestamp;
+    optional binary Type (UTF8);
+    optional binary Hostname (UTF8);
+    optional binary Logger (UTF8);
+    optional binary EnvVersion (UTF8);
+    optional int64 Severity;
+    optional int64 Pid;
+
+    optional binary fields_remote_ip (UTF8);
+    optional binary fields_uaid_hash (UTF8);
+    optional binary fields_message (UTF8);
+    optional boolean fields_error;
+    optional binary fields_user_agent (UTF8);
+    optional binary fields_channelid (UTF8);
+    optional binary fields_message_id (UTF8);
+    optional int64 fields_message_size;
+    optional binary fields_message_source (UTF8);
+    optional binary fields_browser_os (UTF8);
```

Comment:
fields_browser_os and fields_browser_version are being created by line 11 as user agent transforms from `"user_agent"`